tensorflow/python/compiler/tensorrt/trt_convert.py

```diff
@@ -111,70 +110,88 @@ def supported_precision_modes():
+        to be replaced by TRTEngineOp.
+      is_dynamic_op: whether to generate dynamic TRT ops which will build the
+        TRT network and engine at run time. i.e. Since TensorRT version < 6.0
+        does not support dynamic dimensions other than the batch dimension,
+        when the TensorFlow graph has a non-batch dimension of dynamic size,
+        we would need to enable this option. This option should be set to True
+        in TF 2.0.
+      maximum_cached_engines: max number of cached TRT engines for dynamic TRT
+        ops. Created TRT engines for a dynamic dimension are cached. This is
+        the maximum number of engines that can be cached. If the number of
+        cached engines is already at max but none of them supports the input
+        shapes, the TRTEngineOp will fall back to run the original TF subgraph
+        that corresponds to the TRTEngineOp.
+      use_calibration: this argument is ignored if precision_mode is not INT8.
+        If set to True, a calibration graph will be created to calibrate the
+        missing ranges. The calibration graph must be converted to an inference
+        graph by running calibration with calibrate(). If set to False,
+        quantization nodes will be expected for every tensor in the graph
+        (exlcuding those which will be fused). If a range is missing, an error
+        will occur. Please note that accuracy may be negatively affected if
+        there is a mismatch between which tensors TRT quantizes and which
+        tensors were trained with fake quantization.
+      max_batch_size: max size for the input batch. This parameter is only
+        effective when is_dynamic_op=False which is not supported in TF 2.0.
+    """
+    self.rewriter_config_template = rewriter_config_template
+    self.max_workspace_size_bytes = max_workspace_size_bytes
+    self.precision_mode = precision_mode
+    self.minimum_segment_size = minimum_segment_size
+    self.is_dynamic_op = is_dynamic_op
+    self.maximum_cached_engines = maximum_cached_engines
+    self.use_calibration = use_calibration
+    self.max_batch_size = max_batch_size
+
+  def _replace(self,
+               rewriter_config_template=None,
+               max_workspace_size_bytes=None,
+               precision_mode=None,
+               minimum_segment_size=None,
+               is_dynamic_op=None,
+               maximum_cached_engines=None,
+               use_calibration=None,
+               max_batch_size=None,
+    """Set value of class data members only if they are passed as arguments.
+
+    We need this function for backward compatibility with NamedTuple
+    TrtConversionParams which forced users to use the following syntax to
+    set parameters:
+      DEFAULT_TRT_CONVERSION_PARAMS._replace(...)
+    """
+    trt_conversion_params = TrtConversionParams()
+    for k, v in vars().items():
+      if v and (k != "self"):
+        setattr(trt_conversion_params, k, v)
+    return trt_conversion_params
+
```

Comment:
Can this class have a __str__ or __repr__ method so the API pbtxt file doesn't end up with python IDs in it?

Reply:
@alextp I added `__repr__` that returns a string. 

I also updated the pbtxt file. 

I use dict style in the returned string which is different from what we had before for the NamedTuple (brace and colon vs. parenthesis and equal sign). Please let me know if this is fine.